### ISSUE TITLE
Increased urine 8-oxo-7,8-dihydroguanosine

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7033]
**Preferred term label:**
Increased urine 8-oxo-7,8-dihydroguanosine level

**Synonyms**
Increased urinary 8-oxo-7,8-dihydroguanosine
Increased urinary 8-oxo-Gsn level

**Definition (free text, please give PubMed ID)**
An elevated amount of 8-oxo-7,8-dihydroguanosine in the urine.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
613630 
PMID: 31345061  2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
